pyproject.toml

```diff
@@ -82,6 +82,9 @@ docs = [
 roman = [
     "roman_datamodels>=0.17.1",
 ]
+strauss = [
+    "strauss@git+https://github.com/james-trayford/strauss@equal_loudness_normalisation"
```

Comment:
PyPI won't take this. Strauss needs a stable release or you can only install it in `tox.ini`. But if the latter, Jdaviz better still be able to run if strauss is not installed.

I think you also introduced a dependence to `tqdm` but it is not declared here.

Reply:
Ok I reached out to James from the Strauss team to see if we can get a release of Strauss out soon. Should I add `tqdm` to our dependencies if it only is used for Strauss?

Reply:
You should add it under the `strauss` directive here, no? I see you explicitly trying to import it in `cube_listener.py`.

Reply:
p.s. Do we really need it? Isn't it for command line progress bar? But we are a GUI program here, not TUI.

Reply:
That's something else I can bring up with the Strauss team.